style.css

```diff
@@ -321,6 +344,133 @@ textarea {
   transform: translateX(5px);
 }
 
+/* popup window section */
+
+.container {
+  padding: 16px;
+  display: flex;
+  gap: 12px;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 300ms ease-in-out;
+  transform: translate(-50%, -50%) scale(0);
+  background-color: white;
+  border: 1px solid #efe1e6;
+  border-radius: 16px;
+  z-index: 10;
+  max-width: 343px;
+  align-items: flex-start;
+}
+
+.container .mimg {
+  width: 311px;
+  height: 220px;
+}
+
+.container .imgs {
+  width: 311px;
+}
+
+.container .dimg {
+  display: none;
+}
+
+#clsbtn {
+  position: relative;
+  width: 22px;
+  bottom: 205px;
+  left: 128px;
+  cursor: pointer;
+  color: #fff;
+}
+
+.container.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.container li {
+  background: #ebf0ee;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.container h3 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 0;
+  color: #172b4d;
+  width: 259px;
+  text-align: left;
+}
+
+.buttons {
+  display: flex;
+  gap: 12px;
+  align-items: center;
+  width: 311px;
+}
+
+.buttons .live,
+.buttons .source {
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 12px 16px;
+  gap: 12px;
+  background-color: #36b37f;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  height: 27px;
+  color: white;
+}
+
+.overlay {
+  transition: 300ms ease-in-out;
+  background-color: grey;
+  opacity: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  pointer-events: none;
+}
+
+.overlay.active {
+  opacity: 0.5;
+  pointer-events: all;
+}
+
+.errors {
+  display: flex;
+  justify-content: start;
+  color: purple;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+}
```

Comment:
[OPTIONAL]

Consider adding some background color or impactful animation to your error message to make it more noticeable! :fire: